### PR TITLE
[ISTC.js]: Replace (deprecated) for each loops

### PR DESCRIPTION
**Pull request by adomasven**
Also change inconsistent line endings to be in line with the rest of translators using `\n`

### CODE REVIEW

File: ISTC.js [hunk 1]
```diff
@@ -12,127 +12,128 @@
 	"lastUpdated": "2016-11-01 14:02:29"
```
Comment (zuphilip) [inline ISTC.js]: You have to increase the date here, in order to trigger the distribution to the clients.

**Comment by adomasven** [comment]
>It looks that you changed also some whitespaces because the diff shows that everything changed

Yes. This is my IDE automatically converting newline characters. I scanned all translator files and it seems that this is the only one which had windows style newlines `\r\n` mixed in with `\n`. Needless to say, having mixed newlines can cause all kinds of unexpected issues, so all of those were replaced with `\n`, per my commit comment.

**Comment by zuphilip** [comment]
Okay, that looks good for me. 👍 Changing `\r\n` to `\n` is reasonable and consistent, then. (On Windows the default for git is to allow such line endings but to check them into the Linux way.)

**Comment by adam3smith** [comment]
Great, thanks!

**Pull request merged**